### PR TITLE
update changelog for v0.8.0

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

part of #3351

### Description
update changelog

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,53 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.8.0] - 2021-11-25
+### Added
+* Overview of [new features in v0.8](docs/source/whatsnew_0_8.md)
+* Network modules for differentiable neural network topology search (DiNTS)
+* Multi-instance learning transforms and models for digital pathology image analysis
```
Comment (myron) [inline CHANGELOG.md:13]: change to"

Multiple Instance Learning transforms and models for digital pathology WSI analysis

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**